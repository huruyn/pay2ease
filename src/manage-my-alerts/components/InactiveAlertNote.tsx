import { Col, Row } from 'react-bootstrap';

export default function InactiveAlertNote() {
  return (
    <div className="mt-2">
      <hr
        style={{
          marginTop: '3rem',
          height: '2px',
          color: 'C4C4C4',
        }}
      />
      <div className="d-flex">
        <div
          style={{
            width: 20,
            height: 25,
            border: '1px dotted grey',
            textAlign: 'center',
            marginRight: 10,
          }}
        >
          P
        </div>
        <span style={{ fontWeight: 'bold' }}>Note:</span>
      </div>
      <div className="d-flex flex-column">
        <ul>
          <li>
            An inactive alert status applies to bank accounts that were
            automatically transferred from Paperless to eStatements. If your
            alert status is displayed as &apos;Inactive&apos;, it means you have
            not agreed to the terms and conditions for eStatements. Once you
            have accepted the terms and conditions, your alerts will be activated
            and you will not see an &apos;Inactive&apos; alert status.
          </li>
          <li>
            In the case of credit cards, your alerts status will always be
            active.
          </li>
          <li>
            ZippyX Alerts Terms and Conditions are now a part of the
            <a href="/" className="text-black">
              Electronic Access Agreement
            </a>
          </li>
        </ul>
        <Row>
          <Col md={12} className="d-flex align-items-center mt-2">
            <div
              style={{
                width: 20,
                height: 25,
                border: '1px dotted grey',
                textAlign: 'center',
                marginRight: 10,
              }}
            >
              P
            </div>
            <span>
              eStatement Terms and Conditions (last updated: November 2011)
              (PDF, 145KB).
            </span>
          </Col>
        </Row>
      </div>
    </div>
  );
}
