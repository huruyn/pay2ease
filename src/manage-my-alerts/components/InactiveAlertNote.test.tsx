import React from 'react';
import InactiveAlertNote from './InactiveAlertNote';

const ReactTestRenderer = require('react-test-renderer');

describe('InactiveAlertNote Component', () => {
  it('matches the snapshot', () => {
    const tree = ReactTestRenderer.create(
      <InactiveAlertNote />,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
