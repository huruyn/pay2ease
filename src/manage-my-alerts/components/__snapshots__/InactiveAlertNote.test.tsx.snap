// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InactiveAlertNote Component matches the snapshot 1`] = `
<div
  className="mt-2"
>
  <hr
    style={
      Object {
        "color": "C4C4C4",
        "height": "2px",
        "marginTop": "3rem",
      }
    }
  />
  <div
    className="d-flex"
  >
    <div
      style={
        Object {
          "border": "1px dotted grey",
          "height": 25,
          "marginRight": 10,
          "textAlign": "center",
          "width": 20,
        }
      }
    >
      P
    </div>
    <span
      style={
        Object {
          "fontWeight": "bold",
        }
      }
    >
      Note:
    </span>
  </div>
  <div
    className="d-flex flex-column"
  >
    <ul>
      <li>
        An inactive alert status applies to bank accounts that were automatically transferred from Paperless to eStatements. If your alert status is displayed as 'Inactive', it means you have not agreed to the terms and conditions for eStatements. Once you have accepted the terms and conditions, your alerts will be activated and you will not see an 'Inactive' alert status.
      </li>
      <li>
        In the case of credit cards, your alerts status will always be active.
      </li>
      <li>
        ZippyX Alerts Terms and Conditions are now a part of the
        <a
          className="text-black"
          href="/"
        >
          Electronic Access Agreement
        </a>
      </li>
    </ul>
    <div
      className="row"
    >
      <div
        className="d-flex align-items-center mt-2 col-md-12"
      >
        <div
          style={
            Object {
              "border": "1px dotted grey",
              "height": 25,
              "marginRight": 10,
              "textAlign": "center",
              "width": 20,
            }
          }
        >
          P
        </div>
        <span>
          eStatement Terms and Conditions (last updated: November 2011) (PDF, 145KB).
        </span>
      </div>
    </div>
  </div>
</div>
`;
